ui/components/institutional/custody-labels/custody-labels.tsx

```diff
@@ -11,11 +11,7 @@ import {
   TextVariant,
   Display,
 } from '../../../helpers/constants/design-system';
-
-type LabelItem = {
-  key: string;
-  value: string;
-};
+import { LabelItem } from '../../../pages/institutional/custody/custody';
```

Comment:
could we move the interface to a file that's more public accessible as `institutional/custody/types;`